apps/marketplace/components/Brief/Edit/EditOpportunityTable.js

```diff
@@ -0,0 +1,227 @@
+          </div>
+          {showInvited && (
+            <div
+              className={`${styles.greyBorderBottom1} ${styles.paddingTop1} ${styles.paddingBottom1} ${localStyles.editSection}`}
+            >
+              <div>
+                <AUheading className={localStyles.editSectionHeading} level="2" size="sm">
+                  Invited sellers
+                </AUheading>
+                <Link to="/sellers" className={`au-btn au-btn--tertiary ${styles.floatRight} ${localStyles.editLink}`}>
+                  {sellersToInvite.length > 0 ? 'Edit' : 'Add'}
+                </Link>
+              </div>
+              <div>
+                <ul className={`${styles.marginTop0} ${styles.marginBottom0}`}>
+                  {Object.values(brief.sellers).map(seller => (
+                    <li key={seller.name}>{seller.name}</li>
+                  ))}
+                </ul>
+              </div>
+              {sellersToInvite.length > 0 && (
+                <div className={styles.marginTop1}>
+                  <AUheading className={localStyles.editSectionHeading} level="2" size="sm">
+                    Sellers to invite
+                  </AUheading>
+                  <ul className={`${styles.marginTop0} ${styles.marginBottom0}`}>
+                    {sellersToInvite.map(code => (
+                      <li key={edits.sellers[code].name}>{edits.sellers[code].name}</li>
+                    ))}
+                  </ul>
+                </div>
+              )}
+            </div>
+          )}
+          <div
+            className={`${styles.greyBorderBottom1} ${styles.paddingTop1} ${styles.paddingBottom1} ${localStyles.editSection}`}
+          >
+            <div>
+              <AUheading className={localStyles.editSectionHeading} level="2" size="sm">
+                Summary
+              </AUheading>
+              <Link to="/summary" className={`au-btn au-btn--tertiary ${styles.floatRight} ${localStyles.editLink}`}>
+                Edit
+              </Link>
+            </div>
+            <div>
+              <SummaryPreview brief={brief} edits={edits} previewHeight={80} />
+            </div>
+          </div>
+          <div
+            className={`${styles.greyBorderBottom1} ${styles.paddingTop1} ${styles.paddingBottom1} ${localStyles.editSection}`}
+          >
+            <div>
+              <AUheading className={localStyles.editSectionHeading} level="2" size="sm">
+                Closing date
+              </AUheading>
+              <Link
+                to="/closing-date"
+                className={`au-btn au-btn--tertiary ${styles.floatRight} ${localStyles.editLink}`}
+              >
+                Extend
+              </Link>
+            </div>
+            <div>
+              <p>
+                {itemWasEdited(format(new Date(brief.dates.closing_time), 'YYYY-MM-DD'), edits.closingDate)
+                  ? format(edits.closingDate, 'dddd DD MMMM YYYY [at 6pm (in Canberra)]')
```

Comment:
did you test this for timezones?

Reply:
6pm has been hard coded in for both format strings.